### PR TITLE
docs: add page hosting

### PR DESCRIPTION
**Pull request by Caixetadev**

### CODE REVIEW

File: docs/src/content/docs/es/guides/hosting.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+```dockerfile
+FROM alpine:latest
+
+ARG GO_FAST_VERSION=0.1.0
+
+RUN apk add --no-cache unzip openssh
+
+# download and unzip go-fast-cdn
+ADD https://github.com/kevinanielsen/go-fast-cdn/releases/download/${GO_FAST_VERSION}/go-fast-cdn-x86_64-linux.zip /tmp/cdn.zip
```
Comment (kevinanielsen) [inline docs/src/content/docs/es/guides/hosting.md:43]:
```suggestion
ADD https://github.com/kevinanielsen/go-fast-cdn/releases/download/${GO_FAST_VERSION}/go-fast-cdn_${GO_FAST_VERSION}_linux_amd64.zip /tmp/cdn.zip
linux.zip /tmp/cdn.zip
```

File: docs/src/content/docs/es/guides/hosting.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+```bash
+touch Dockerfile
+```
+
+A partir de ahí, debes pegar las siguientes líneas en el Dockerfile, que copiarán el binario de go-fast-cdn diseñado para distribuciones de Linux y lo servirán en el puerto 8080.
+
+```dockerfile
+FROM alpine:latest
+
+ARG GO_FAST_VERSION=0.1.0
```
Comment (kevinanielsen) [inline docs/src/content/docs/es/guides/hosting.md:38]:
```suggestion
ARG GO_FAST_VERSION=0.1.6
```

**Review by kevinanielsen — CHANGES_REQUESTED** [review]

**Review by kevinanielsen — APPROVED** [review]
LGTM - Thank you!

**Comment by Caixetadev** [comment]
You're welcome

**Pull request merged**